site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,448 @@
+import Image from '../../../components/Image';
+import InternalLink from '../../../components/InternalLink';
+import {
+  Breadcrumb,
+  Button,
+  Header,
+  Hero,
+  Navigation,
+  SideNavigation,
+  StatusLabel,
+  Tabs,
+  Footer,
+  IconFacebook,
+  IconTwitter,
+  IconInstagram,
+  IconYoutube,
+  IconTiktok,
+} from 'hds-react';
+import PlaygroundPreview from '../../../components/Playground';
+
+# Building navigation
+
+<StatusLabel variant="rounded" type="error">Draft</StatusLabel>
+
+<LeadParagraph>
+  Navigation components are crucial as they help users navigate through an application or website seamlessly.
+</LeadParagraph>
+
+## Principles
+
+Adhering to these principles can create an effective and user-friendly navigation component that enhances the overall user experience.
+
+<div
+  class="guideline-do"
+  style="background: var(--color-success-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-do-label" style="color:var(--color-success); margin-bottom:var(--spacing-s);">
+    <IconCheck style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Do</strong>
+  </div>
+  <ul style="margin-bottom:0;">
+    <li>
+      <strong>Keep it as simple as possible.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Consider the information architecture and site hierarchy. Determine what the most crucial information is and how many levels are necessary.
+        </li>
+      </ul>
+    </li>
+    <li>
+      <strong>Pay attention to label lengths and links.</strong>
+      <ul style="margin-top:0;">
+        <li>Keep them as concise as possible and ensure navigation links match their destination pages. If multiple links lead to the same page, use identical labels.</li>
+      </ul>
+    </li>
+    <li>
+      <strong>Prioritise mobile users.</strong>
+      <ul style="margin-top:0;">
+        <li>Design your navigation system with a 'mobile-first' mindset and ensure seamless functionality on mobile devices and touchscreens.</li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+<div
+  class="guideline-dont"
+  style="background: var(--color-error-light); padding: var(--spacing-s); margin-bottom: var(--spacing-layout-2-xs); max-width:600px;"
+>
+  <div class="guideline-dont-label" style="color:var(--color-error); margin-bottom:var(--spacing-s);">
+    <IconError style={{ marginRight: 'var(--spacing-3-xs)', verticalAlign: 'middle' }} size="s" />
+    <strong>Don't</strong>
+  </div>
+  <ul style="margin-bottom:0;">
+    <li>
+      <strong>Don't utilize the content area for navigation purposes.</strong>
+      <ul style="margin-top:0;">
+        <li>
+          Avoid duplicating all navigation links in the content area; use content-relevant navigation links on specific pages.
+        </li>
+        <li>
+          Exception: On landing pages consider utilizing the content area for navigation for mobile usage.
```

Comment:
Maybe I would phrase it slighlty differently and tell also why mobile users benefit from this.
E.g.
"On landing pages it is possible to provide navigation links to support better mobile experience. Many mobile users do not open the menus, so links in content area help them to reach the needed content."

Reply:
Good addition. I'll edit it.